samples/coop_synchronization/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _coop_synchronization_sample:
+
+Cooperative Synchronization Sample
+##################################
+
+Overview
+********
+
+A simple application that demonstrates even more basic sanity of the kernel.
+Two threads (A and B) take turns printing a greeting message to the console.
+They use yields and semaphores to control the whose turn it is to print.
+The rate is only controlled via busy waits. This demonstrates kernel communication only.
+This is a good example to test new platforms that may not have a working clock.
+I.e. this will still work when :kconfig:option:`CONFIG_SYS_CLOCK_EXISTS` =n
+It's also a very simplistic demo allowing users to follow the execution trace in
+a no-timer (i.e. nonexistent, disabled, deferred) scenario and experience predictability.
+To that end, sample should ideally be built with no interrupt-driven-UART to avoid spurious
+IRQ fires which may trigger unpredictable context switch. However, some platforms do not
+support polling api for UART. For those, only the regular variant is built.
+
+Building and Running
+********************
+
+This project outputs to the console.  It can be built and executed
+on QEMU as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/coop_synchronization
+   :host-os: unix
+   :board: qemu_x86
+   :goals: run
+   :compact:
+
+Sample Output
+=============
+
+.. code-block:: console
+
+   thread_a: level 1 irq_lock key = 534
+   thread_a: level 2 irq_lock key = 22
```

Comment:
As we don't print it anymore - let's drop it from documentation as well.

Reply:
Fixed!